tests/service/test_app_state.py

```diff
@@ -0,0 +1,13 @@
+"""Tests for the app state module."""
+
+from src.service import app_state
+
+
+def test_app_state_imports():
+    """Test that app_state module can be imported."""
+    assert app_state is not None
```

Comment:
for all these import tests, if the module can't be imported the parent module will fail, so they don't really do anything

Reply:
yea. Those are all dummy tests. Just to make sure the imported module is used so my linter doesn't flag it.

Reply:
I see, You could also probably note it as an unused import but whatever